### DISCUSSION TITLE
LazyRoute does not have values like `to` or `fullPath` similar to Route

### DISCUSSION THREAD

**Discussion by svihpinc-off** [TanStack/router#2917] [Ideas]
when create a lazy route using `createLazyFileRoute`, it results in a `LazyRoute` type object. But unlike the `Route` type from `createFileRoute`, `LazyRoute` does not provide values like `to` or `fullPath` to be used in, eg, `Link`.

```ts
// about.lazy.tsx
const Route = createLazyFileRoute('/about')({component: About});

// contact.tsx
const Route = createFileRoute('/contact')({component: Contact});

// index.tsx
import {Route as aboutRoute} from './about.lazy';
import {Route as contactRoute} from './contact';

// ...
<Link to={aboutRoute.to} /> // invalid
<Link to={contactRoute.to} /> // valid
```

Could we get the same values under `LazyRoute` as are under `Route` type?

**Comment by schiller-manuel** [comment]
with automatic code splitting you don't really need lazy routes anymore. does this solve your issue?
